devices/models.py

```diff
@@ -215,6 +215,8 @@ class Lending(models.Model):
     device = models.ForeignKey(Device, null=True, blank=True)
     smalldevice = models.CharField(_("Small Device"), max_length=200, null=True, blank=True)
 
+reversion.register(Lending, ignore_duplicates = True)
```

Comment:
apparently we can use decorators now, and it's sufficient to just import reversion:

http://django-reversion.readthedocs.io/en/stable/api.html

Reply:
changed it where there were no extra options, but imo 
   " If you’ve registered your models using reversion.register(), the admin class will use the     configuration you specify there. Otherwise, the admin class will auto-register your model, following all inline model relations and parent superclasses." menas that extra options have to be/ can be specified like before?

Reply:
But there is no admin class here?